hack/run-tests.sh

```diff
@@ -54,5 +54,5 @@ echo "Running test suite..."
 $CONTAINER_CLI run $background_args \
   -v $SCRIPT_DIR/../website/docs:/content \
   -v $SCRIPT_DIR/../manifests:/manifests \
-  -e 'EKS_CLUSTER_NAME' -e 'AWS_REGION' \
+  -e 'EKS_CLUSTER_NAME' -e 'AWS_REGION' -e 'C9_USER' \
```

Comment:
Is this designed to cater for the code being developed in Cloud9? If I'm developing on my local laptop will this skip triggering the Cloud9 logic in the rest of the change?

Reply:
I needed it to emulate the C9 deployment in my local computer.  But might be unnecessary, I'll do a test and remove it if it is not necessary.

Reply:
Variable removed